From 307bc6b8237e934b5dbd650ecaa273d35c186497 Mon Sep 17 00:00:00 2001
From: Markus Armbruster <armbru@redhat.com>
Date: Wed, 31 Mar 2010 13:24:28 -0300
Subject: [PATCH 13/66] tools: Remove unused cur_mon from qemu-tool.c

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1270041921-28969-14-git-send-email-armbru@redhat.com>
Patchwork-id: 8237
O-Subject: [PATCH 13/66] tools: Remove unused cur_mon from qemu-tool.c
Bugzilla: 579470
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>

(cherry picked from commit 070a2f819a2ec03f26d6033f522f0ce9e29f6539)
---
 qemu-tool.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 qemu-tool.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/qemu-tool.c b/qemu-tool.c
index 18b48af..bbce99f 100644
--- a/qemu-tool.c
+++ b/qemu-tool.c
@@ -33,8 +33,6 @@ void qemu_service_io(void)
 {
 }
 
-Monitor *cur_mon;
-
 void monitor_printf(Monitor *mon, const char *fmt, ...)
 {
 }
-- 
1.7.0.3

